package.json

```diff
@@ -5,26 +5,27 @@
   },
   "dependencies": {
     "@bitgo-beta/sdk-coin-near": "^1.1.1-beta.45",
-    "@bitgo/abstract-utxo": "^1.1.1",
-    "@bitgo/sdk-api": "^1.1.1",
-    "@bitgo/sdk-coin-avaxc": "^2.0.0-rc.4",
-    "@bitgo/sdk-coin-bch": "^1.1.1",
-    "@bitgo/sdk-coin-bcha": "^1.4.0",
-    "@bitgo/sdk-coin-bsv": "^1.1.1",
-    "@bitgo/sdk-coin-btc": "^1.0.2",
-    "@bitgo/sdk-coin-btg": "^1.1.1",
-    "@bitgo/sdk-coin-dash": "^1.1.1",
-    "@bitgo/sdk-coin-dot": "^1.1.0",
-    "@bitgo/sdk-coin-eos": "^1.0.2",
-    "@bitgo/sdk-coin-eth": "^1.1.1",
-    "@bitgo/sdk-coin-ethw": "^1.1.0-rc.19",
-    "@bitgo/sdk-coin-ltc": "^1.1.1",
-    "@bitgo/sdk-coin-polygon": "^1.1.0-rc.33",
-    "@bitgo/sdk-coin-sol": "^2.0.0",
-    "@bitgo/sdk-coin-trx": "^1.0.2",
-    "@bitgo/sdk-coin-xlm": "^1.0.2",
-    "@bitgo/sdk-coin-xrp": "^1.1.1",
-    "@bitgo/sdk-coin-zec": "^1.1.1",
+    "@bitgo/abstract-utxo": "^1.10.0",
+    "@bitgo/sdk-api": "^1.9.1",
+    "@bitgo/sdk-coin-avaxc": "^2.3.1",
+    "@bitgo/sdk-coin-bch": "^1.3.6",
+    "@bitgo/sdk-coin-bcha": "^1.5.1",
+    "@bitgo/sdk-coin-bsv": "^1.3.6",
+    "@bitgo/sdk-coin-btc": "^1.2.6",
+    "@bitgo/sdk-coin-btg": "^1.3.6",
+    "@bitgo/sdk-coin-dash": "^1.3.6",
+    "@bitgo/sdk-coin-doge": "^1.8.1",
+    "@bitgo/sdk-coin-dot": "^2.0.1",
+    "@bitgo/sdk-coin-eos": "^1.3.6",
+    "@bitgo/sdk-coin-eth": "^2.6.0",
+    "@bitgo/sdk-coin-ethw": "^1.3.6",
+    "@bitgo/sdk-coin-ltc": "^2.0.1",
+    "@bitgo/sdk-coin-polygon": "^1.3.6",
+    "@bitgo/sdk-coin-sol": "^2.3.0",
+    "@bitgo/sdk-coin-trx": "^1.3.1",
+    "@bitgo/sdk-coin-xlm": "^1.2.6",
+    "@bitgo/sdk-coin-xrp": "^1.3.6",
+    "@bitgo/sdk-coin-zec": "^1.3.6",
```

Comment:
Had to update most of the bitgo coin dependencies to resolve this issue: https://bitgo.slack.com/archives/C010L370H3N/p1669744852296769